src/components/Budget/BudgetView/BudgetViewContainer.js

```diff
@@ -74,6 +82,32 @@ const BudgetViewContainer = ({ history, match, mutator }) => {
     [history, budget],
   );
 
+  const removeBudget = useCallback(
+    () => {
+      mutator.budgetById.DELETE({ id: budgetId })
+        .then(() => {
+          showCallout({ messageId: 'ui-finance.budget.actions.remove.success', type: 'success' });
+          history.replace({
+            pathname: `${FUNDS_ROUTE}/view/${budget.fundId}`,
+            search: location.search,
+          });
+        })
+        .catch(() => {
+          showCallout({ messageId: 'ui-finance.budget.actions.remove.error', type: 'error' });
```

Comment:
IMHO, that's not fulfill the ticket's requirement. Budget could not be deleted by different reasons.

Reply:
Fixed. Anyway, I'm going to add error code for deletion after [MODFIN-117](https://issues.folio.org/browse/MODFIN-117) is done

Reply:
Added code: `transactionIsPresentBudgetDeleteError` from [MODFIN-117](https://issues.folio.org/browse/MODFIN-117) description